File: 800/Sum/sum.c
// https://codeforces.com/problemset/problem/1742/A


#include <stdio.h>

// Main driver
int main() {
    int n, i;
    int arr[3];
    printf("Enter the test cases: ");
    scanf("%d", &n);
    while(n > 0) {
        for(i = 0; i < 3; i++) {
            printf("Enter the data: ");
            scanf("%d", &arr[i]);
        }
        printf("The array is: ");
        for(i = 0; i < 3; i++) {
            printf("%d " " ", arr[i]);
        }
        printf("\n");            //cout << endl;
        if(arr[0] + arr[1] == arr[2] || arr[0] + arr[2] == arr[1] || arr[1] + arr[2] == arr[0]) {
            printf("YES.\n");
        }
        else {
            printf("NO.\n");;
        }

        printf("\n");
        n--;
    }
    return 0;
}

File: 800/Sum/sum.py
# https://codeforces.com/problemset/problem/1742/A

n = int(input("Enter the number of test cases: "))
print("Test cases are: ", n)

while n > 0:
  a = []
  for i in range(0, 3):
    l = int(input())
    a.append(l)

  print("The array is: ", a)
  if a[0] + a[1] == a[2] or a[0] + a[2] == a[1] or a[1] + a[2] == a[0]:
    print("YES.")
  else:
    print("NO.")

  n = n - 1
